doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,182 @@
+
+Each of the following levels on the list also includes the previous one.
+See :ref:`zigbee_ug_logging_stack_logs` to read more about trace logs.
+
+
+.. rst-class:: numbered-step
+
+Configure how to print ZBOSS trace logs
+***************************************
+
+The :ref:`zigbee_osif_zboss_osif_serial` offers a few backends to choose from for printing ZBOSS trace logs.
+It is recommended to use the Zigbee serial logger, as it is the most efficient.
+To enable it, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING`.
+
+Optional
+   You can increase size of the ring buffer that temporarily stores the trace logs.
+   To do this, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE`.
+   This can prevent losing some of the logs in demanding scenarios such as high network traffic, multiple devices being configured or joined, and so on.
+
+
+Trace logs using UART (default)
+===============================
+
+Trace logs using the UART are enabled by default.
+Below are more details on how to configure trace logs using the UART.
+
+1. Set with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_UART_LOGGING`.
+
+#. Optionally, configure which UART device you want to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   The default ``UART_1`` will be used if no other UART device is configured.
+
+#. Configure the UART device that you want to use to be connected to the onboard J-Link instead of ``UART_0``, by extending the DTS overlay file for the selected board with the following:
+
+    .. code-block:: devicetree
+
+      &uart1 {
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+      &uart0 {
+         rx-pin = <33>;
+         tx-pin = <34>;
+         /delete-property/ rts-pin;
+         /delete-property/ cts-pin;
+      };
+
+    .. note::
+      This will remove access to the application logs.
+
+
+Optional configuration
+-----------------------
+
+You can also increase the UART throughput by changing the baudrate.
+Some of the trace logs will be dropped if the throughput is too low.
+By default, the UART baudrate is set to 115200.
+To increase the baudrate to ``1000000``, add the ``current-speed = <1000000>;`` property to the ``uart1`` node in the DTS overlay file.
+This can be done like the following:
+
+.. code-block:: devicetree
+
+      &uart1 {
+         current-speed = <1000000>;
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+
+.. _ug_zigbee_configuring_zboss_traces_using_usb:
+
+Trace logs using native USB
+===========================
+
+Trace logs can also be configured using a native USB.
+This is useful because console logs can still be read through USB, and trace logs that are printed through the UART cannot.
+This is because the UART used by the console logs are not connected to JLink.
```

Comment:
```suggestion
This is useful because trace logs will be printed through separate virtual COM port so the console logs can still be read through J-Link COM port
```